test/fixtures/subnets.yml

```diff
@@ -7,6 +7,7 @@ one:
   dhcp: one
   tftp: two
   dns: three
+  bmc: bmc
```

Comment:
the BMC smart proxy is not assigned to organization and location so under user, it will fail to load by `host.subnet.bmc`. Apply the following to resolve this

```diff
diff --git a/test/fixtures/taxable_taxonomies.yml b/test/fixtures/taxable_taxonomies.yml
index d5078ef8e..dc595a353 100644
--- a/test/fixtures/taxable_taxonomies.yml
+++ b/test/fixtures/taxable_taxonomies.yml
@@ -203,3 +203,13 @@ puppetmaster_organization2:
   taxonomy: organization2
   taxable: puppetmaster
   taxable_type: "SmartProxy"
+
+bmc_organization:
+  taxonomy: organization1
+  taxable: bmc
+  taxable_type: "SmartProxy"
+
+bmc_location1:
+  taxonomy: location1
+  taxable: bmc
+  taxable_type: "SmartProxy"
```